adhocracy4/polls/static/PollDetail/PollQuestions.jsx

```diff
@@ -239,25 +247,28 @@ class PollQuestions extends React.Component {
 
   handleSubmit (e) {
     e.preventDefault()
+
     this.setState({
       loading: true,
       checkedTermsOfUse: false
     })
-    const modifiedQuestions = this.state.questions.filter(q => q.modified)
-    const validatedQuestions = modifiedQuestions.filter(q => {
-      if (!q.is_open) {
-        const otherChoice = q.choices.find(c => c.is_other_choice)
-        const otherChoiceSelected = otherChoice && q.userChoices.filter(uc => uc === otherChoice.id).length > 0
+
+    const modifiedAnswers = this.state.questions.filter(question => question.modified)
+
+    const validatedQuestions = modifiedAnswers.filter(question => {
+      if (!question.is_open) {
```

Comment:
thought using words was clearer specially in case of userChoice but happy to go with majority on this?

Reply:
yes, it is better :)